src/handlers/flow/FlowStreamHandler.cpp

```diff
@@ -684,6 +688,9 @@ void FlowMetricsBucket::specialized_merge(const AbstractMetricsBucket &o, [[mayb
                     top_dir.second.topSrcIPPort.merge(interface.second->directionTopN.at(top_dir.first).topSrcIPPort);
                     top_dir.second.topDstIPPort.merge(interface.second->directionTopN.at(top_dir.first).topDstIPPort);
                 }
+                if (group_enabled(group::FlowMetrics::TopTos)) {
```

Comment:
we should probably name things as "DSCP" instead of "TOS" (or "ToS"). We might want to implement actual TOS metrics in the future, or ECN metrics.

Reply:
done